src/stylesheets/components/_sidebar.scss

```diff
@@ -0,0 +1,81 @@
+.sidebar {
+  position: fixed;
+  right: 0;
+  z-index: 1;
+
+  margin: 2rem 2rem 0;
+
+  &__user-image {
+    width: rem(36px);
+    height: rem(36px);
+
+    border-radius: 50%;
+
+    cursor: pointer;
+  }
+
+  &__nav-menu {
+    position: fixed;
+    top: 0;
+    right: 0;
+
+    justify-content: center;
+    width: 0;
+    height: 100%;
+    overflow: hidden;
+
+    background-color: map-get($brand-black, 'lighter');
+
+    backdrop-filter: blur(rem(82px));
+
+    transition: 0.5s;
+  }
+
+  &__nav-menu-active {
+    width: rem(264px);
+  }
+
+  &__nav-menu-items {
+    width: 100%;
+    margin: rem(32px) 0;
+    padding: 0;
+
+    list-style: none;
+  }
+
+  &__nav-menu-information {
+    display: flex;
+    align-items: center;
+    height: rem(36px);
+    margin: 0 rem(32px) rem(20px);
```

Comment:
@rafayet-monon Should we move this margin left and right to `&__nav-menu-items`? so the line inside the sidebar looks the same as the design
![Screen Shot 2564-01-06 at 10 35 24](https://user-images.githubusercontent.com/1772999/103726398-6f791280-500b-11eb-9c4b-b33584dd6a9f.png)
![Screen Shot 2564-01-06 at 10 36 55](https://user-images.githubusercontent.com/1772999/103726415-7869e400-500b-11eb-9744-dc0e84a41eaa.png)